.github/workflows/build.yml

```diff
@@ -9,14 +9,21 @@ jobs:
   build:
     runs-on: ubuntu-18.04
     steps:
-    - name: checkout
-      uses: actions/checkout@v2
-
+    
     - uses: actions/setup-go@v2
       with:
         go-version: 1.16
 
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: copy working directory to GOPATH
```

Comment:
Why is this needed? The checkout doesn't put the files in the right place?

Reply:
this was not needed, it's the way follow in another place (maybe putting here in same commit ya confusing). But we have removed this build action so we are good.